models/invitationCodes.js

```diff
@@ -0,0 +1,45 @@
+InvitationCodes = new Mongo.Collection('invitation_code');
```

Comment:
Mongo collection should match the variable name: `invitation_codes` (plural).

Also, determine whether underscore or camel case should be used for multiple words.